### PR TITLE
docs: add Chinese translation for the new brotil plugin

### LINKED ISSUE #10811: docs: add Chinese translation for the new `brotil` plugin
### Current State

Feature added in this PR and will be included in the 3.8.0 release: https://github.com/apache/apisix/pull/10515

English doc currently available here: https://apisix.apache.org/docs/apisix/next/plugins/brotli

### Desired State

Should have the corresponding Chinese doc as well.

### PR DESCRIPTION
**Pull request by smileby**
### Description

add Chinese translation for the new brotil plugin

Fixes #10811 

### Checklist

- [X] I have explained the need for this PR and the problem it solves
- [X] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [X] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/brotli.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+| mode         | integer              | False | 0             | [0, 2]       | 动态设置 `brotli decompress mode`，更多信息参考 [RFC 7932](https://tools.ietf.org/html/rfc7932)                                          |
+| lgwin        | integer              | False | 19            | [0, 10-24]   | 动态设置 `Brotli滑动窗口大小`，`lgwin` 是滑动窗口大小的以 2 为底的对数，将其设置为 0 会让压缩器自行决定最佳值，更多信息请参考 [RFC 7932](https://tools.ietf.org/html/rfc7932)    |
+| lgblock      | integer              | False | 0             | [0, 16-24]   | 动态设置 `Brotli输入块大小`，`lgblock` 是最大输入块大小的以 2 为底的对数，将其设置为 0 会让压缩器自行决定最佳值，更多信息请参考 [RFC 7932](https://tools.ietf.org/html/rfc7932)	 |
+| http_version | number               | False | 1.1           | 1.1, 1.0     | 与 `gzip_http_version` 指令类似，用于识别 http 的协议版本                                                                                    |
+| vary         | boolean              | False | false         |              | 与 `gzip_vary` 指令类似，用于启用或禁用 `Vary: Accept-Encoding` 响应头                                                                        |
+
+## 启用插件
+
+如下示例中，在指定的路由上启用 `brotli` 插件
```
Comment (kayx23) [inline docs/zh/latest/plugins/brotli.md:65]:
```suggestion
如下示例中，在指定的路由上启用 `brotli` 插件：
```

File: docs/zh/latest/plugins/brotli.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        "nodes": {
+            "httpbin.org": 1
+        }
+    }
+}'
+```
+
+## 使用示例
+
+通过上述命令启用插件后，可以通过以下方法测试插件。
```
Comment (kayx23) [inline docs/zh/latest/plugins/brotli.md:86]:
```suggestion
通过上述命令启用插件后，可以通过以下方法测试插件：
```

File: docs/zh/latest/plugins/brotli.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+| types        | array[string] or "*" | False | ["text/html"] |              | 动态设置 `brotli_types` 指令。特殊值 `"*"` 用于匹配任意的 MIME 类型                                                                              |
+| min_length   | integer              | False | 20            | >= 1         | 动态设置 `brotli_min_length` 指令。指定进行压缩的文件最小的长度                                                                                    |
+| comp_level   | integer              | False | 6             | [0, 11]      | 动态设置 `brotli_comp_level` 指令                                                                                                   |
+| mode         | integer              | False | 0             | [0, 2]       | 动态设置 `brotli decompress mode`，更多信息参考 [RFC 7932](https://tools.ietf.org/html/rfc7932)                                          |
+| lgwin        | integer              | False | 19            | [0, 10-24]   | 动态设置 `Brotli滑动窗口大小`，`lgwin` 是滑动窗口大小的以 2 为底的对数，将其设置为 0 会让压缩器自行决定最佳值，更多信息请参考 [RFC 7932](https://tools.ietf.org/html/rfc7932)    |
```
Comment (kayx23) [inline docs/zh/latest/plugins/brotli.md:58]: image

mistakenly google translated?

**Review by kayx23 — CHANGES_REQUESTED** [review]

File: docs/zh/latest/plugins/brotli.md (hunk 3 above)
Comment (smileby) [reply docs/zh/latest/plugins/brotli.md:58]: There is no need for translation here, right?

Comment (smileby) [reply docs/zh/latest/plugins/brotli.md:58]: > image
> 
> mistakenly google translated?

I plan to change it to this

```text
动态设置 `brotli_min_length` 指令。
动态设置 `brotli_comp_level` 指令。
动态设置 `brotli decompress mode`，更多信息参考 [RFC 7932](https://tools.ietf.org/html/rfc7932) 
动态设置 `brotli sliding window size`，`lgwin` 是滑动窗口大小的以 2 为底的对数，将其设置为 0 会让压缩器自行决定最佳值，更多信息请参考 [RFC 7932](https://tools.ietf.org/html/rfc7932) 
动态设置 `brotli input block size`，`lgblock` 是最大输入块大小的以 2 为底的对数，将其设置为 0 会让压缩器自行决定最佳值，更多信息请参考 [RFC 7932](https://tools.ietf.org/html/rfc7932)
```

do you have a better idea

Comment (kayx23) [reply docs/zh/latest/plugins/brotli.md:58]: I don't expect anything wrapped in inline code to be translated.

However in this case, the original was `brotli input block size`, which, I don't know why it was inline coded initially. I think it actually makes sense to update the English doc to remove the inline code style for these phrases.

Comment (kayx23) [reply docs/zh/latest/plugins/brotli.md:58]: > I plan to change it to this
> 
> ```
> 动态设置 `brotli_min_length` 指令。
> 动态设置 `brotli_comp_level` 指令。
> 动态设置 `brotli decompress mode`，更多信息参考 [RFC 7932](https://tools.ietf.org/html/rfc7932) 
> 动态设置 `brotli sliding window size`，`lgwin` 是滑动窗口大小的以 2 为底的对数，将其设置为 0 会让压缩器自行决定最佳值，更多信息请参考 [RFC 7932](https://tools.ietf.org/html/rfc7932) 
> 动态设置 `brotli input block size`，`lgblock` 是最大输入块大小的以 2 为底的对数，将其设置为 0 会让压缩器自行决定最佳值，更多信息请参考 [RFC 7932](https://tools.ietf.org/html/rfc7932)
> ```

@smileby I'm good with this too.

**Review by kayx23 — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Pull request merged**